packages/create-plugin/src/commands/generate.command.ts

```diff
@@ -1,15 +1,318 @@
-import path from 'path';
+// import path from 'path';
```

Comment:
This is getting long and harder to follow. Let's create a `generate` directory and put the prompt part in a promptGenerate.ts file?